docs/docs/migration.md

```diff
@@ -51,12 +51,53 @@ You need to update your `mta.yaml` file to exclude `html5` modules from the resu
         properties:
           url: ${default-url}
 ```
-   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; After renaming, make sure that the   places where the name is used refer to the correct artifact.
-
-   
+After renaming, make sure that the places where the name is used refer to the correct artifact.
   
 ---
 **NOTE**
 If you try to build the project without these changes, the build tool will identify this and will fail the build with the corresponding errors.
 
 ---
+
+* `JSON` files with [service creation parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) or [service binding parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) referenced by `path` property of the correponding entity in the `mta.yaml` are packaged differently into the result MTA archive. 
```

Comment:
The service creation parameters and service binding parameters point to the same link, is this intentional?

Reply:
Fixed